QUESTIONS.md

```diff
@@ -1,5 +1,10 @@
 # Questions about Code, Node, Express, React
 
+1) In retrospect, I would like to review some of the higher order functions, and have more indeep coverage on this part.
+
 What questions can you come up with?
 
 ## Cohort Questions
+
+Are we getting any grades in order to determine how we are doing 
```

Comment:
I was wondering the same thing recently!

Reply:
Same! Ive been thinking of making a website that grabs student's repos from githubs and allows Stack to apply grades directly to github repos and allows students to login and see their grades, then donating it to Slack when I get done with the program. This is for sure an area of imporvement